modules/accredited_representatives/config/routes.rb

```diff
@@ -2,5 +2,6 @@
 
 AccreditedRepresentatives::Engine.routes.draw do
   namespace :v0, defaults: { format: :json } do
+    post 'power_of_attorney/accept', to: 'power_of_attorney#accept'
```

Comment:
Will this possibly become a member action on a collection resourceful route later?